learn/advanced/known_limitations.md

```diff
@@ -112,3 +112,15 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
 **Limitation:** Meilisearch might throw an internal error when indexing large batches of documents.
 
 **Explanation:** Indexing a large batch of documents, such as a JSON file over 3.5GB in size, can result in Meilisearch opening too many file descriptors. Depending on your machine, this might reach your system's default resource usage limits and trigger an internal error. Use [`ulimit`](https://www.ibm.com/docs/en/aix/7.1?topic=u-ulimit-command) or a similar tool to increase resource consumption limits before running Meilisearch. For example, call `ulimit -Sn 3000` in a UNIX environment to raise the number of allowed open file descriptors to 3000.
+
+## Maximum database size
+
+**Limitation:** The maximum size of an index is 500GiB, and the maximum size of the task database is 10GiB.
+
+**Explanation:** Meilisearch allocates all the virtual memory it requires upfront. The database maximum size ensures instances can contain several large indexes without reaching OS limits on the amount of virtual memory available to a single process.
+
+## Maximum number of indexes in an instance
+
+**Limitation:** A single Meilisearch instance can safely contain around 180 indexes in Linux and macOS environments, and around 15 indexes in Windows environments.
```

Comment:
@dureuill: I extrapolated "15 indexes in Windows" from your comments on how Windows has a much lower virtual memory per process limit and how that gives us less wiggle room. Do suggest a different number if you have anything more accurate. I do feel it's good to give users a rough idea of a safe-ish limit on Windows, though.

Reply:
5 indexes of 500GB leaves 2.5TB of additional leeway so should be fine I think.